components/parental_control/extension/service_worker/background.js

```diff
@@ -160,14 +182,20 @@ const logoutUser = async (password, sendResponse) => {
     }
 }
 
-const handleBrowserWindows = async (url) => {
-    const windows = await chrome.windows.getAll({ populate: true })
-    windows.forEach((window) => {
-        chrome.windows.remove(window.id);
-    });
-    if (!url) await chrome.windows.create({ type: 'normal' });
-    else chrome.windows.create({ url: url, type: 'normal' });
-}
+const handleBrowserWindows = async (url) => {    
+    let newWindow;
+    if(url === 'extension/content/ui/sessionTimeout.html')
```

Comment:
make this url ('extension/content/ui/sessionTimeout.html') a constant

Reply:
and instead of passing a url you can just pass a flag for to show session timed out screen or not